pallets/xyk/src/tests/liquidity.rs

```diff
@@ -185,6 +185,34 @@ fn add_liquidity_as_another_user_should_work() {
 	});
 }
 
+#[test]
+fn add_liquidity_should_work_when_limit_exceeds_account_balance() {
```

Comment:
this does not sound right.  if a limit is exceeded - stuff should not work. 

perhaps just better wording here. 

i would like to see a test where it fails.

Reply:
Renamed.
The test where it fails is [here](https://github.com/galacticcouncil/Basilisk-node/blob/7a1bddd4670c7dcc4b14489b8b5799685f91246d/pallets/xyk/src/tests/liquidity.rs#L395)